src/modules/catalog.js

```diff
@@ -1758,9 +1769,9 @@ class Catalog {
    *         pattern: 'outerwear',
    *         match_type: 'EXACT'
```

Comment:
Should `match_type` also be changed to be camelCase? 🤔 I'm not sure how far we want to take this

Reply:
🤔 Let's leave it at surface level parameters that are documented by us, not anything under it.

Reply:
Sounds like we want to change this to be camel cased 👍